shared/js/textChatView.jsx

```diff
@@ -87,21 +90,32 @@ loop.shared.views.chat = (function(mozL10n) {
         );
       }
 
+      // Only render messages after they've come back from the server to avoid
+      // dups in a simple way for early product standup.
+      //  XXX akita Probably not the UX we want; should talk to UX folks.
+      if (this.props.type === CHAT_MESSAGE_TYPES.SENT) {
+        return null;
```

Comment:
Won't this cause all messages to appear blue as they're all RECEIVED now? I suppose that might be part of the XXX comment. Could you file the bug and put the bug number here?